reference/component-versions.md

```diff
@@ -2,6 +2,40 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated, this is called out in the [release notes](https://discourse.ubuntu.com/t/release-notes/17842) and components are marked with `n/a` below.
 
+## 1.22.2
+
+### Charms
+
+N/A
+
+### Bundles
+
+N/A
+
+### Snaps
+
+| Name |  Channel | Version |
+|----------|--------------|---------|
+| `ams`    | `1.22/stable` | 1.22.2 |
+| `aar`    | `1.22/stable` | 1.22.2 |
+| `ams-node-controller` | `1.22/stable` | 1.22.2 |
+| `anbox-cloud-dashboard` | `1.22/stable` | 1.22.2 |
+| `anbox-stream-agent` | `1.22/stable` | 1.22.2 |
+| `anbox-stream-gateway` | `1.22/stable` | 1.22.2 |
+| `anbox-cloud-appliance` | `1.22/stable` | 1.22.1 |
```

Comment:
One small question: do we need to list the snap like anbox-cloud-appliance here if there is no update for it in one patch release?

Reply:
Wouldn't it be useful to know which version of snap we are on? I don't mind removing it either.